scripts/e2e.truffle.sh

```diff
@@ -16,13 +16,32 @@ git clone https://github.com/trufflesuite/truffle.git
 cd truffle
 yarn bootstrap
 
+yarn config set registry http://localhost:4873
+
+echo ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
+echo "Updating @truffle/interface-adapter"
+echo ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
+
+# @truffle/interface-adapter
+cd packages/interface-adapter
```

Comment:
Is there a way to have the uninstalling and installing process for ``@truffle/interface-adapter`` and ``@truffle/contract`` managed here in a more generic way? 

It will probably be the case that also other modules will get dependencies added to web3 packages and it would be great to have this for example in a well-structured and easy to read JSON object.

Reply:
The best way would be upgrade web3 uniformly across all packages at Truffle. At truffle theydo this using a 3rd party lerna utility called lerna-update-wizard which looks like [does not support][https://github.com/Anifacted/lerna-update-wizard/issues/36] specifying a private registry for the new dependency when using yarn.

But I agree this is tightly coupled to a specific configuration of dependencies truffle/contract currently has and could need maintenance in the future.

Perhaps this question can be addressed a later PR. Honestly not sure how to resolve at the moment and the current commands achieve the near term goal... need to think about this more though.

Reply:
> it would be great to have this for example in a well-structured and easy to read JSON object.

Oh, just coming back here for a second look - this is a really good suggestion. Would need to add logic for processing the JSON in shell, maybe even with a node script. 

Can we investigate systematically updating all the web3 deps at Truffle in a second PR and if that's not possible or too complicated - implementing this idea?

Reply:
Sounds good 👌